vm/test.c

```diff
@@ -192,6 +192,17 @@ static void *readfile(const char *path, size_t maxlen, size_t *len)
     return data;
 }
 
+#ifndef memfrob
```

Comment:
Is memfrob defined as a macro? That would surprise me. How about checking for `__GLIBC__` instead?